analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -200,6 +212,30 @@ if (!dir.exists(results_dir)) {
   dir.create(results_dir)
 }
 
+# Read in GISTIC `broad_values_by_arm.txt` file
+gistic_df <-
+  data.table::fread(unzip(
+    file.path(root_dir, "data", "pbta-cnv-cnvkit-gistic.zip"),
+    files = file.path(
+      "2019-12-10-gistic-results-cnvkit",
+      "broad_values_by_arm.txt"
+    ),
+    exdir = file.path(root_dir, "scratch")
+  ))
+
+# Read in UCSC cytoband data
+ucsc_cytoband <-
+  data.table::fread(
+    "http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz"
+  )
```

Comment:
Do you need `data.table = FALSE` here? I am unsure.

Reply:
It ran fine without the argument but I will still include it.